doc/examples/applications/plot_human_mitosis.py

```diff
@@ -0,0 +1,170 @@
+"""
+================================
+Segment human cells (in mitosis)
+================================
+
+In this example, we analyze a microscopy image of human cells. We use data
+provided by Jason Moffat [1]_ through
+[CellProfiler](https://cellprofiler.org/examples/#human-cells).
+
+.. [1] Moffat J, Grueneberg DA, Yang X, Kim SY, Kloepfer AM, Hinkle G, Piqani B, Eisenhaure TM, Luo B, Grenier JK, Carpenter AE, Foo SY, Stewart SA, Stockwell BR, Hacohen N, Hahn WC, Lander ES, Sabatini DM, Root DE (2006) "A lentiviral RNAi library for human and mouse genes applied to an arrayed viral high-content screen" Cell, 124(6):1283-98. DOI: [10.1016/j.cell.2006.01.040](https://doi.org/10.1016/j.cell.2006.01.040). PMID: 16564017
```

Comment:
For the DOI, please use the notation  

>   :DOI:`1234` 

Please, wrap at 80 chars also.

Reply:
Oh, thanks! I should have done a recursive grep for "DOI" in `./doc/examples`... ;) Fixed.